### PR TITLE
Note in README, and an image

### PR DESCRIPTION
**Pull request by mmikhasenko**
Follow up of #73.

I did not figure out how to upload to `https://user-images.githubusercontent.com/`,
linked the one from the PR #73  `https://github.com/user-attachments`

### CODE REVIEW

File: image.png
Comment (BeastyBlacksmith) [inline image.png:1]: That shouldn't be part of the repo, I'd say

Comment (mmikhasenko) [reply image.png:1]: What do you refer to, the image, or note in README.md?

Comment (BeastyBlacksmith) [reply image.png:1]: I refer to the image file

Comment (mmikhasenko) [reply image.png:1]: Do you have suggestions where the image file should go.

I aimed to use `user-images.githubusercontent.com` URLs, as for other images in the README, but GitHub now defaults to github.com/user-attachments/assets URLs in most uploads. Unfortunately, I do not find any to force the old URL structure.

The user-attachments links should be stable. Let me know if you’d prefer any adjustments here!

**Comment by mmikhasenko** [comment]
@BeastyBlacksmith please note a few new lines of instructions in the `README.md`.
Default image is of different size, the note on `dpi` hopefully will help the future people

File: image.png
Comment (BeastyBlacksmith) [reply image.png:1]: Its not about the link. That is fine. You just shouldn't commit the image itself to the repo

Comment (mmikhasenko) [reply image.png:1]: ah damn 💩 , sure

**Pull request merged**